moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -108,20 +119,24 @@ export const formatProjectActivity = (change, lookupList) => {
         },
       ],
     };
-  }
-  // we dont need the lookup table
-  else {
-    // If the update is an object, show just the field name that was updated.
+  } else {
+    // If the update is an object, check first for a null object
     if (typeof changeData.new[changedField] === "object") {
-      return {
-        changeIcon,
-        changeText: [
-          {
-            text: `Changed ${entryMap.fields[changedField]?.label} `,
-            style: null,
-          },
-        ],
-      };
+      // if the new field is null, it is because something was removed
+      if (lookupList[changeData.new[changedField]] == null) {
```

Comment:
i think its preferred to use === where possible in javascript

Reply:
yup, fix before merging, nice catch Rose.

Reply:
ack embarrassing typo thanks for catching that